2021-02-25 15:06:04,432 INFO    MainThread:13877 [wandb_setup.py:_flush():69] setting env: {}
2021-02-25 15:06:04,432 INFO    MainThread:13877 [wandb_setup.py:_flush():69] setting login settings: {}
2021-02-25 15:06:04,432 INFO    MainThread:13877 [wandb_setup.py:_flush():69] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f89421a4150>
2021-02-25 15:06:04,432 INFO    MainThread:13877 [wandb_init.py:_log_setup():319] Logging user logs to /home/dacon/Dacon/bdg/ct/segment_abdomen/src/wandb/run-20210225_150604-3sf85rlw/logs/debug.log
2021-02-25 15:06:04,432 INFO    MainThread:13877 [wandb_init.py:_log_setup():320] Logging internal logs to /home/dacon/Dacon/bdg/ct/segment_abdomen/src/wandb/run-20210225_150604-3sf85rlw/logs/debug-internal.log
2021-02-25 15:06:04,432 INFO    MainThread:13877 [wandb_init.py:init():352] calling init triggers
2021-02-25 15:06:04,432 INFO    MainThread:13877 [wandb_init.py:init():359] wandb.init called with sweep_config: {}
config: {'path': '../data/normal2', 'name': 'timm-efficientnet-b7', 'model': 'timm-efficientnet-b7', 'pretrain': 'noisy-student', 'model_path': '../models/timm-efficientnet-b7', 'augmentation': [], 'num_epochs': 20, 'batch_size': 4, 'optimizer': <class 'torch.optim.adam.Adam'>, 'scheduler': <class 'torch.optim.lr_scheduler.CosineAnnealingWarmRestarts'>, 'loss': <class 'segmentation_models_pytorch.utils.losses.DiceLoss'>, 'seed': 1015, 'device': 'cuda:0'}
2021-02-25 15:06:04,433 INFO    MainThread:13877 [wandb_init.py:init():374] re-initializing run, found existing run on stack: 3kp7epzr
2021-02-25 15:06:04,434 INFO    MainThread:13877 [wandb_run.py:finish():1078] finishing run deagwon-bu/ct_segment/3kp7epzr
2021-02-25 15:06:04,434 INFO    MainThread:13877 [wandb_init.py:_jupyter_teardown():250] cleaning up jupyter logic
2021-02-25 15:06:04,434 INFO    MainThread:13877 [jupyter.py:save_history():208] not saving jupyter history
2021-02-25 15:06:04,434 INFO    MainThread:13877 [wandb_run.py:_atexit_cleanup():1392] got exitcode: 0
2021-02-25 15:06:04,434 INFO    MainThread:13877 [wandb_run.py:_restore():1364] restore
2021-02-25 15:06:04,749 INFO    MainThread:13877 [wandb_run.py:_wait_for_finish():1515] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 854
  total_bytes: 854
}

2021-02-25 15:06:06,762 INFO    MainThread:13877 [wandb_run.py:_wait_for_finish():1515] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5042
  total_bytes: 5042
}

2021-02-25 15:06:08,766 INFO    MainThread:13877 [wandb_run.py:_wait_for_finish():1515] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5042
  total_bytes: 5042
}

2021-02-25 15:06:10,085 INFO    MainThread:13877 [wandb_run.py:_show_files():1737] logging synced files
2021-02-25 15:06:10,092 INFO    MainThread:13877 [wandb_init.py:init():401] starting backend
2021-02-25 15:06:10,093 INFO    MainThread:13877 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-02-25 15:06:10,098 INFO    MainThread:13877 [backend.py:ensure_launched():123] starting backend process...
2021-02-25 15:06:10,138 INFO    MainThread:13877 [backend.py:ensure_launched():128] started backend process with pid: 14006
2021-02-25 15:06:10,139 INFO    MainThread:13877 [wandb_init.py:init():406] backend started and connected
2021-02-25 15:06:10,150 INFO    MainThread:13877 [wandb_init.py:init():446] updated telemetry
2021-02-25 15:06:10,151 INFO    MainThread:13877 [wandb_init.py:init():465] communicating current version
2021-02-25 15:06:10,731 INFO    MainThread:13877 [wandb_init.py:init():470] got version response 
2021-02-25 15:06:10,731 INFO    MainThread:13877 [wandb_init.py:init():478] communicating run to backend with 30 second timeout
2021-02-25 15:06:11,065 INFO    MainThread:13877 [wandb_init.py:init():503] starting run threads in backend
2021-02-25 15:06:12,064 INFO    MainThread:13877 [wandb_run.py:_console_start():1422] atexit reg
2021-02-25 15:06:12,065 INFO    MainThread:13877 [wandb_run.py:_redirect():1285] redirect: SettingsConsole.WRAP
2021-02-25 15:06:12,065 INFO    MainThread:13877 [wandb_run.py:_redirect():1320] Wrapping output streams.
2021-02-25 15:06:12,065 INFO    MainThread:13877 [wandb_run.py:_redirect():1336] Redirects installed.
2021-02-25 15:06:12,065 INFO    MainThread:13877 [wandb_init.py:init():527] run started, returning control to user process
2021-02-25 15:06:54,999 INFO    MainThread:13877 [wandb_run.py:_atexit_cleanup():1392] got exitcode: 0
2021-02-25 15:06:55,000 INFO    MainThread:13877 [wandb_run.py:_restore():1364] restore
2021-02-25 15:06:55,303 INFO    MainThread:13877 [wandb_run.py:_wait_for_finish():1515] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 854
  total_bytes: 854
}

2021-02-25 15:06:57,314 INFO    MainThread:13877 [wandb_run.py:_wait_for_finish():1515] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4737
  total_bytes: 4737
}

2021-02-25 15:06:58,634 INFO    MainThread:13877 [wandb_run.py:_show_files():1737] logging synced files
